### PR TITLE
New package: burp 1.4.40

### PR DESCRIPTION
**Pull request by yopito**
backup and restore program with a client/server architecture. 
Storage is disk based, simplier than bacula.
Home Page: http://burp.grke.net

### CODE REVIEW

**Comment by skrzyp** [comment]
Please rebase :)

File: srcpkgs/burp-server/template [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+build_style=gnu-configure
+configure_args="--sysconfdir=/etc/burp
+--sbindir=/usr/bin"
+conf_files="/etc/burp/burp-server.conf"
+
+post_install() {
+	vsv ${pkgname}
+
+	# same files than client package
+	vdoc "README"
```
Comment (xtraeme) [inline srcpkgs/burp-server/template]: These files, except `README` and `UPGRADING` are just irrelevant for us, please drop them.

File: srcpkgs/burp-server/template [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	sed -e "s,/usr/sbin/burp_ca,/usr/bin/burp_ca,g" -i ${DESTDIR}/etc/burp/burp-server.conf
+
+	# storage folder
+	touch -f ${PKGDESTDIR}/var/spool/burp/.empty_on_purpose
+}
+
+burp-client_package() {
+	short_desc="A network-based backup and restore program - client"
+	# openssl binary needed by burp_ca script
+	depends="libressl-openssl"
```
Comment (xtraeme) [inline srcpkgs/burp-server/template]: `libressl-openssl` -> `openssl` instead (it's a virtual package).

File: srcpkgs/burp-server/template [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	conf_files="/etc/burp/burp.conf"
+	pkg_install() {
+		vmove "usr/bin/burp"
+		vmove "usr/bin/burp_ca"
+		vmove "usr/share/man/man8/burp.8.gz"
+		vmove "usr/share/man/man8/burp_ca.8.gz"
+		vmove "etc/burp/CA-client"
+		vmove "etc/burp/burp.conf"
+		vdoc "README"
```
Comment (xtraeme) [inline srcpkgs/burp-server/template]: Ditto, do not install useless files unnecessarily.

**Comment by yopito** [comment]
Thanks for your inputs.
Regarding "useless files": documentation is sometime difficult to find with burp, that's why I push hard to keep it into packages ...
Alternative: create a dedicated sub-package -doc ?

**Comment by xtraeme** [comment]
Seriously there's no need for a `-doc` subpkg to contain these files... they should not be shipped by the binary packages.

**Comment by yopito** [comment]
OK ... I will strip down my package and rebase it as requested.

**Comment by yopito** [comment]
On 09/17/15 07:16, Juan RP wrote:
> Merged #2519 <https://github.com/voidlinux/void-packages/pull/2519>.

Thanks !
My first package ;)
Pierre

**Pull request merged**